sunpy/lightcurve/sources/eve.py

```diff
@@ -36,7 +36,8 @@ class EVELightCurve(LightCurve):
 
     Level 0CS data is primarily used for space weather. It is provided near
     real-time and is crudely calibrated 1-minute averaged broadband irradiances
-    from ESP and MEGS-P broadband.
+    from ESP and MEGS-P broadband. For other levels of EVE data, use
+    sunpy.net.Fido, with sunpy.net.attrs.Instrument('eve').
```

Comment:
Should these not be `~sunpy.net.Fido` and ``sunpy.net.attrs.Instrument('eve')``?

Reply:
You're saying to add a tilde and change "with" to "and"?

Reply:
Sorry, I left a confusing and incomplete comment. 

I think it should be:
```
`~sunpy.net.Fido`
```
so that it creates a link to the Fido docs and 
```
`sunpy.net.attrs.Instrument('eve')`
```
so that it is highlighted from the background.